life_expectancy/cleaning.py

```diff
@@ -28,28 +43,23 @@ def clean_data(country):
     dataframe["year"] = pd.to_numeric(dataframe["year"], errors="coerce").astype(int)
 
     # Apply the str_to_float method and perform data cleaning
-    dataframe['value'] = [str_to_float(val) for val in dataframe['value']]
+    dataframe['value'] = [__str_to_float(val) for val in dataframe["value"]]
 
     # Remove natural NaNs, but also the coerced invalid values from the previous value conversion
     dataframe.dropna(subset=["value"], inplace=True)
 
     # Filter by region
     dataframe = dataframe[dataframe["region"] == country]
+    
+    return dataframe
 
-    # Save transformed data
-    dataframe.to_csv("life_expectancy/data/pt_life_expectancy.csv", index = False)
+def save_data(dataframe: pd.DataFrame, save_path: str) -> None:
```

Comment:
**Nitpick:** since we might want to use this variable to save other data in the same path, but not with the same name, I would give as argument to this function, instead of the path to the file, the name of the file we want to save. 
Anyways you can interpret the function to be used to save any file in any path, so depends a lot on your interpretation of the function

Reply:
Yes! é o que já estou a fazer na verdade, deves estar a comentar sobre o código a vermelho que foi apagado do commit anterior :)